### ISSUE TITLE
[$15]The form fields on the page do not have associated labels.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3232]
**Describe the bug**

The form fields on the page do not have associated labels.

**To Reproduce**

1. Go to Members login:https://accounts.topcoder.com/member. 
2. Type in each form field to see if there is a visible label which remains at all times.
3. Notice that the placeholder text disappears as soon as we start typing in the form fields.
readers.

**Expected behavior**

There should be a visible label at all times which is associated with the respective form field.

**Actual behavior**

The placeholder text has been given for all the form fields which disappears when user starts typing.

**Additional context**
Type:Form Fields

Page: https://accounts.topcoder.com/member

Status: Partially fixed, provide aria-label as 'user name or email' since this is the text given as a placeholder. The password field doesn't have an associated label, provide aria-label for it too.

**Comment by PrakashDurlabhji** [comment]
@veshu will it be open soon?

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099920 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099920 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099920 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099920<br/><br/>```This is an automated message for crazyk via Topcoder X```
